src/planner/binder/statement/bind_update.cpp

```diff
@@ -98,9 +101,35 @@ BoundStatement Binder::Bind(UpdateStatement &stmt) {
 	auto &table_binding = (BoundBaseTableRef &)*bound_table;
 	auto table = table_binding.table;
 
-	auto root = CreatePlan(*bound_table);
-	auto &get = (LogicalGet &)*root;
-	assert(root->type == LogicalOperatorType::GET);
+	if (stmt.from_table) {
+		auto bound_from = Bind(*stmt.from_table);
+		unique_ptr<BoundCrossProductRef> bound_crossproduct;
+		bool bound_left = false;
+		if (bound_from->type == TableReferenceType::CROSS_PRODUCT) {
```

Comment:
What is the reason for this special case? Can't you just always create a cross product between bound_from and bound_table?

Reply:
Duh, of course. Will fix.